.eleventy.js

```diff
@@ -16,6 +16,10 @@ export default async function(eleventyConfig) {
     ],
   });
 
+    eleventyConfig.addPassthroughCopy({
+      'test/_fonts': 'fonts',
```

Comment:
oh, I didn't think about how we actually ship the fonts as part of the plugin. This is only providing them for the development test setup. I guess there would be a few options:

- We could include them in the package, and somehow document how to copy them into your project
- We could include them from google fonts by default (`@import url('https://fonts.googleapis.com/css2?family=Recursive&display=swap');`) 
- We could document adding them yourself

Reply:
Ok, I've been looking into the different approaches. 

If we include fonts in the package that people could copy into the project, would that be something like listing the location of the fonts in the  ["exports" field](https://nodejs.org/api/packages.html#exports) in `package.json`? And users could import those if they choose?

Would adding fonts in a separate layer make sense?

Reply:
@mirisuzanne I ended up removing the fonts, thinking that route was the simplest. We can just document how people can add their own fonts. I can start adding that to the documentation.